flink-ai-flow/ai_flow/store/db/db_model.py

```diff
@@ -124,13 +137,13 @@ class SqlModelVersionRelation(base):
 
     version = Column(String(255), primary_key=True)
```

Comment:
`version` field shoud not be regarded as `primary_key`.

Reply:
```model_id = Column(BigInteger, ForeignKey('model_relation.uuid', onupdate='cascade'), primary_key=True)```
model_id is specified as primary_key as well, so the model_id and version would be regard as Composite Primary Key in sqlalchemy.